front/app/components/admin/PostManager/components/PostTable/Row/IdeaRow.tsx

```diff
@@ -219,17 +220,21 @@ const IdeaRow = ({
         );
       },
     },
-    {
-      name: 'down',
-      Component: ({ idea }: IdeaCellComponentProps) => {
-        return (
-          <>
-            <Icon name="thumbs down" />
-            {idea.attributes.dislikes_count}
-          </>
-        );
-      },
-    },
+    ...(type !== 'ProjectProposals'
```

Comment:
We don't allow down voting for proposals so we don't need this column in the case of proposals.